### PR TITLE
method EQL for comp

### PR DESCRIPTION
**Pull request by alessandrosimon**
Competition Checklist:
- [x] title of this PR is meaningful, i.e. "method X for comp"
- [x] A folder has been added to `submission/` with a meaningful name corresponding to your method name.
- The added folder includes these elements:
    - [x] `metadata.yml` (**required**): A file describing your submission, following the descriptions in `example/metadata.yml`.  
    - [x] `regressor.py` (**required**): a Python file that defines your method, named appropriately. See [submission/feat-example/regressor.py](https://github.com/cavalab/srbench/blob/Competition2022/submission/feat-example/regressor.py) for complete documentation.  It contains:
        - [x]  `est`: a sklearn-compatible `Regressor` object. 
        - [x]  `model(est, X=None)`: a function that returns a [**sympy-compatible**](https://www.sympy.org) string specifying the final model. It can optionally take the training data as an input argument. See [guidance below](###-returning-a-sympy-compatible-model-string). 
        - [x]  `eval_kwargs` (optional): a dictionary that can specify method-specific arguments to `evaluate_model.py`.
        
    - [x] `LICENSE` *(optional)* A license file
    - [x] `environment.yml` *(optional)*: a [conda environment file](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-from-an-environment-yml-file) that specifies dependencies for your submission. 
    - [x] `install.sh` *(optional)*: a bash script that installs your method. 
    - [x] additional files *(optional)*: you may include a folder containing the code for your method in the submission. 

I have verified that:

- [x] install scripts do not require sudo permissions. 
- [x] if pulled remotely, the source code is a fixed version (i.e., rerunning `install.sh` shouldn't pulll a different version of the code when run multiple times.) 

Refer to the [competition guide]( https://cavalab.org/srbench/competition-guide/ ) if you are unsure about any steps. 
If you don't find an answer, ping us!

### CODE REVIEW

**Comment by alessandrosimon** [comment]
The build is failing but unfortunately I'm not really sure what the problem is. 
Does anyone know more?

**Comment by lacava** [comment]
i'm not sure what changed exactly, but on a couple PRs I am seeing this issue. I updated the base branch to include pyyaml. Please rebase to the current version and see if that fixes it.

File: submission/eql/regressor.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    model_str = str(est.best_estimator_.get_eqn())
+
+    if isinstance(X, pd.DataFrame):
+        mapping = {"x" + str(i): k for i, k in enumerate(X.columns)}
+        for k, v in mapping.items():
+            model_str = model_str.replace(k, v)
+
+    return model_str
+
+
```
Comment (lacava) [inline submission/eql/regressor.py:31]: might I suggest you add a `test_params` setting to shorten the tests? something like 

```python
test_params={'param_grid':{'n_iter':[10]}}
```

Comment (alessandrosimon) [reply submission/eql/regressor.py:31]: Good idea, just added it.

**Pull request merged**